GFS fixed-file YAMLs.

HenryRWinterbottom opened the pull request:
**Description**

This PR addressed issue #1470. 

**Type of change**

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

**How Has This Been Tested?**

No testing required; new `parm` level files only.
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

HenryRWinterbottom committed:
GFS fixed-file YAMLs.

aerorahul reviewed (approved):
looks good

aerorahul commented on parm/ufs/fix/gfs/atmos.fixed_files.yaml:
```diff
@@ -0,0 +1,87 @@
+- [$(FIX_lut)/optics_SU.v1_3.dat,  $(DATA)/optics_SU.dat]
+
+# fd_nems.yaml file
+- [$(HOMEgfs)/sorc/ufs_model.fd/tests/parm/fd_nems.yaml, $(DATA)/]
```
This needs to be indented.

aerorahul reviewed (changes requested):
just discovered that the atmos fix files need to be indented.

HenryRWinterbottom committed:
Update atmos.fixed_files.yaml

Indented.

FWIW, YAML is smart enough to not require the indents; it looks for the `:` to define the keys.

aerorahul reviewed (approved):
looks good.
Just note that you have 2 different indentations going.
`atmos.fixed_files.yaml` is using 2 spaces while the other are using 4.
